src/ResourceManager/Network/Commands.Network/help/Update-AzureRmVpnSite.md

```diff
@@ -0,0 +1,419 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/update-azurermvpnsite
+schema: 2.0.0
+---
+
+# Update-AzureRmVpnSite
+
+## SYNOPSIS
+Updates a VpnSite representing a customer branch to an intended goal state.
+
+## SYNTAX
+
+### ByVpnSiteName (Default)
+```
+Update-AzureRmVpnSite -ResourceGroupName <String> -Name <String> [-VirtualWanResourceGroupName <String>]
+ [-VirtualWanName <String>] [-VirtualWan <PSVirtualWan>] [-VirtualWanId <String>] [-IpAddress <String>]
```

Comment:
If you want to support multiple ways to obtain VirtualWan, you will need to add a set of parameter sets for each.

Reply:
Added parameter sets and updated help module.